packages/cli/src/index.js

```diff
@@ -0,0 +1,57 @@
+#! /usr/bin/env node
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict
+ */
+
+import yargs from 'yargs';
+import path from 'path';
+import chalk from 'chalk';
+
+import { isDir, makeCompiledDir } from './files';
+import { compileDirectory } from './transform';
+import options from './options';
+import errors from './errors';
+import watch from './watch';
+
+console.log(
+  chalk.green(`\n
+  ______   _             _         __   __
+  / ____| | |           | |        \\ \\ / /
+ | (___   | |_   _   _  | |   ___   \\ V / 
+  \\___ \\  | __| | | | | | |  / _ \\   > <  
+  ____) | | |_  | |_| | | | |  __/  / . \\ 
+ |_____/   \\__|  \\__, | |_|  \\___| /_/ \\_\\
+                  __/ |                   
+                 |___/                
+`),
```

Comment:
This doesn't looks particularly readable

Reply:
There's some escaped characters. I'll attach a screenshot of what it looks like in the CLI.

Reply:
<img width="632" alt="image" src="https://github.com/facebook/stylex/assets/24259636/b058acc0-559d-466e-b40e-abd1e31e0fe4">